eng/templates/stages/oav.yml

```diff
@@ -0,0 +1,140 @@
+extends:
+  template: /eng/templates/stages/1es-redirect.yml
+  parameters:
+    stages:
+      - stage: Build_And_Test
+        displayName: Build and Test
+
+        variables:
+          - template: /eng/templates/variables/image.yml
+
+        jobs:
+          - job: Build
+
+            pool:
+              name: $(LINUXPOOL)
+              image: $(LINUXVMIMAGE) 
+              os: linux
+
+            steps:
+              - checkout: self
+                submodules: true
+
+              - task: NodeTool@0
+                inputs:
+                  versionSpec: "20.x"
+                displayName: "Use Node 20"
```

Comment:
It would be good to test on a sparse matrix of Linux/Windows and Node18/Node20, like I do here:

https://github.com/Azure/azure-rest-api-specs/blob/bf9738223ed8c1c25485980b74cfb8ef4158b847/.github/workflows/_reusable-eng-tools-test.yaml#L15

Reply:
I agree. Tests crash on windows currently, and I don't want to deal with that at this time,